include/dkm.hpp

```diff
@@ -38,6 +38,22 @@ T distance_squared(const std::array<T, N>& point_a, const std::array<T, N>& poin
 	return d_squared;
 }
 
+/*
+ Calculate the mean square float distance between a collection of points.
+ */
+template <typename T, size_t N>
+float point_collection_epsilon(const std::vector< std::array<T, N> >& point_a, const std::vector< std::array<T, N> >& point_b) {
```

Comment:
I know i proposed this. But it seems to have a fundamental error, as with the initially forked terminating condition. My bad, for not checking it more.

Examining if two `std::vector<...>` objects are different is not correct, as even having the same elements with a different order, renders the two as different. There is simply no 1-1 correlation between the two sets of centers. 

As @FoteiniFotopoulou suggested, we could simply:

1. calculate the two mean points of `point_a` and `point_b`
2. calculate their difference and return it as the result of this method

To calculate the mean of each collection, use the calculations in `details::calculate_means`.